### PR TITLE
ONYX-14082 - Push sequel to the latest version

### PR DESCRIPTION
**Pull request by sashaCher**
### Desired Outcome

Conjur is using latest version of the `sequel` gem.

### Implemented Changes

- Version pinning has been removed from the Gemfile for `sequel` gem
- `20160628222441_create_credentials.rb` is changed following some errors during db migration.
  Now it's aligned with primary key creation in `20160628212347_create_roles.rb` file.
  DB structure remains the same
-  `config/application.rb` is running at the beginning db migration task. It's trying to initialize `:pg_hstore` db extension that does not exist before db migration. It's created as a part of `20180410092453_policy_log.rb` db migration task. So `:pg_hstore` db extension initialization during application init stage is unnecessary, either way it's initialized in `app/models/policy_log.rb` file - the only model is using the extension.

### Connected Issue/Story

[ONYX-14082](https://ca-il-jira.il.cyber-ark.com:8443/browse/ONYX-14082)

### Definition of Done

- [X] Desired outcome has been achieved

#### Changelog

- [ ] The CHANGELOG has been updated, or
- [X] This PR does not include user-facing changes and doesn't require a
  CHANGELOG update

#### Test coverage

- [ ] This PR includes new unit and integration tests to go with the code
  changes, or
- [x] The changes in this PR do not require tests

The changes are tested in CE, see `sequel-upgrade` branch pipeline.

#### Documentation

- [ ] Docs (e.g. `README`s) were updated in this PR
- [ ] A follow-up issue to update official docs has been filed here: [insert issue ID]()
- [X] This PR does not require updating any documentation

#### Behavior

- [ ] This PR changes product behavior and has been reviewed by a PO, or
- [ ] These changes are part of a larger initiative that will be reviewed later, or
- [X] No behavior was changed with this PR

#### Security

- [ ] Security architect has reviewed the changes in this PR,
- [ ] These changes are part of a larger initiative with a separate security review, or
- [X] There are no security aspects to these changes

### CODE REVIEW

**Comment by semyon-estrin** [comment]
LGTM

File: db/migrate/20160628222441_create_credentials.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
   change do
     create_table :credentials do
       # Not an FK, because credentials won't be dropped when the RBAC is rebuilt
-      primary_key :role_id, type: String, null: false
```
Comment (tzheleznyak) [inline db/migrate/20160628222441_create_credentials.rb:7]: null:false in not needed in newer version?

**Review by tzheleznyak — APPROVED** [review]
LGTM

File: db/migrate/20160628222441_create_credentials.rb (hunk 1 above)
Comment (sashaCher) [reply db/migrate/20160628222441_create_credentials.rb:7]: `primary_key: true` includes not null :-)

**Comment by semyon-estrin** [comment]
LGTM

**Review by semyon-estrin — APPROVED** [review]
LGTM

**Pull request merged**